kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerState.java

```diff
@@ -66,6 +66,7 @@
   private final Queue<ConsumerRecord<KafkaKeyT, KafkaValueT>> consumerRecords = new ArrayDeque<>();
 
   volatile long expiration;
+  private final Long expirationLock;
```

Comment:
Added a separate final object for synchronizing the expiration field.

Reply:
Can you just do `Object expirationLock = new Object()` instead? We don't actually need a `Long` for anything.

Reply:
@rigelbm No problem. I'll fix it soon based on your comment.